.github/workflows/labels.yml

```diff
@@ -7,19 +7,14 @@ name: "Label PR"
 
 on:
   workflow_call:
-    inputs:
-      caller:
-        description: Name of the calling workflow.
-        required: true
-        type: string
   workflow_run:
     workflows:
       - Review dismissed
       - Review submitted
     types: [completed]
 
 concurrency:
-  group: ${{ inputs.caller }}-${{ github.workflow }}-${{ github.event_name }}-${{ github.head_ref || github.run_id }}
+  group: Labels-${{ github.workflow }}-${{ github.event_name }}-${{ github.head_ref || github.run_id }}
```

Comment:
Do we want a comment clarifying that the seeminly repeated workflow name is actually intended. I could imagine a PR in the future removing `Lables-` string because it _should_ be the same as `${ github.workflow }}` :grin: 

My wording is horrible, but maybe you could come up with something better:

> Hardcode `Labels` because `github.workflow` is the calling workflow, which may or may not be a different workflow

On the other hand, this may not really be needed, as it is documented by git-blame.